.gitignore

```
@@ -72,6 +72,7 @@ _test_est
 *_local/
 *_local.*
 
+**/__sharrowcache__
```

```suggestion
**/__sharrowcache__
**/skims.zarr
```
In my test runs, skims.zarr is being picked up  by git for placeholder_sandag/output_1